### PR TITLE
APP-3965 Close Modal on 'ESC'

### PR DESCRIPTION
**Pull request by symphony-jean-michael**
**Jira ticket**
https://perzoinc.atlassian.net/browse/APP-3965

**Change**
Close Modal on 'ESC'

### CODE REVIEW

**Comment by symphony-jean-michael** [comment]
ok to test

File: spec/components/modal/Modal.spec.tsx [hunk 1]
```diff
@@ -31,4 +38,26 @@ describe('Modal', () => {
...
+
+    const eventMock = {
+      key: Keys.ESC,
+    };
+
+    // 'ESC' key in the Modal
+    fireEvent.keyUp(title, eventMock);
+
+    // Modal should be closed
+    expect(show).toBeFalsy();
```
Comment (AliciaSymphony) [inline spec/components/modal/Modal.spec.tsx:61]: To have a comparison before/after in your test, I suggest to add either : 
- expect(title).toBeFalsy(); (line 62)
- or expect(show).toBeTruthy(); (line 52)

Comment (symphony-jean-michael) [reply spec/components/modal/Modal.spec.tsx:62]: Ok.Done

**Review by AliciaSymphony — APPROVED** [review]

**Review by symphony-annaLlorens — APPROVED** [review]
LGTM

**Pull request merged**